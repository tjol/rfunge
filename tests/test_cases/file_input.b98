1100"detcepxe.89b.tupni_elif"i v      @,k4"ERROR"
"                          "    <v
"                          "0  <^   a
                               >,v
>d2*aga-;@,,,"ERR";# _@        |:<
^i"file_input.b98.expected"01a0<
