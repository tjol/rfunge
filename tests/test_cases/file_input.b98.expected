This is the test file for 
file_input.b98 